4.3/example.md

```diff
@@ -0,0 +1,83 @@
+    Configuration Example
+
+```sh
+
+docker run -it \
+
+	#Sets the flatnode file, which is to reduce the load on the database when you plan to use multiple countrys together bigger than 6GB
```

Comment:
Please indent with 2 spaces like the other readme.

Reply:
done!